### PR TITLE
trudnowato

### PR DESCRIPTION
**Pull request by VladyslavKanarkin**
everything is possible)

### CODE REVIEW

File: home_work_4/comparsion.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+a1 = input("Text_num1:")
+b1 = input("Text_num2:")
```
Comment (MarinaZelenska) [inline home_work_4/comparsion.py:2]: Could you please use a more informative name of variables?

File: home_work_4/converter.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+
+EXCHANGE_RATES = {
+    "EUR": 50.1,
+    "USD": 45.1,
+    "UAN": 30.1
+}
+
+def check(currency):
+    if currency not in EXCHANGE_RATES: raise ValueError("Неизвестная валюта " + currency)
+
```
Comment (MarinaZelenska) [inline home_work_4/converter.py:10]: I am not sure that this code should be in a separate function

File: home_work_4/converter.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+def convert(curr_from, curr_to, amount):
+    check(curr_from); check(curr_to)
+    return amount * EXCHANGE_RATES[curr_from] / EXCHANGE_RATES[curr_to]
+
+cf = input('Currency1:')
+ct = input('Currency2:')
+amt = int(input('Quantity:'))
+
+print(convert(cf, ct, amt))
```
Comment (MarinaZelenska) [inline home_work_4/converter.py:19]: I am not sure that it logically works correctly
image

File: home_work_4/delete_symbol.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
+string = input("Text:")
+symbol = input("")
```
Comment (MarinaZelenska) [inline home_work_4/delete_symbol.py:2]: Do we need "" inside input() if we don't write nothing?

File: home_work_4/converter.py
Comment (VladyslavKanarkin) [inline home_work_4/converter.py:1]: I'm training to use GitHUb)I couldn’t fix the first option, so I copied it

File: home_work_4/delete_symbol.py
Comment (VladyslavKanarkin) [inline home_work_4/delete_symbol.py:1]: delete " "

**Review by VladyslavKanarkin — COMMENTED** [review]
I couldn’t fix the first option, so I copied it) I'm training to use GitHub)

**Review by MarinaZelenska — APPROVED** [review]

**Comment by VladyslavKanarkin** [comment]
123

**Pull request merged**